spec.html

```diff
@@ -24895,7 +24895,8 @@ <h1>Function.prototype.toString ( )</h1>
         <emu-alg>
           1. Let _func_ be the *this* value.
           1. If _func_ is a <emu-xref href="#sec-bound-function-exotic-objects">Bound Function exotic object</emu-xref> or a <emu-xref href="#sec-built-in-function-objects">built-in function object</emu-xref>, then return an implementation-dependent String source code representation of _func_. The representation must have the syntax of a |NativeFunction|. Additionally, if _func_ is a <emu-xref href="#sec-well-known-intrinsic-objects">Well-known Intrinsic Object</emu-xref> and is not identified as an anonymous function, the portion of the returned String that would be matched by |PropertyName| must be the initial value of the `name` property of _func_.
-          1. If Type(_func_) is Object and _func_ has a [[SourceText]] internal slot and Type(_func_.[[SourceText]]) is String and ! HostHasSourceTextAvailable(_func_) is *true*, then return _func_.[[SourceText]].
+          1. If Type(_func_) is Object and _func_ has a [[SourceText]] internal slot and _func_.[[SourceText]] is a sequence of Unicode code points and ! HostHasSourceTextAvailable(_func_) is *true*, then
+            1. Return the String value consisting of the UTF16Encoding of each code point of _func_.[[SourceText]].
```

Comment:
I think you should call out the concatenation happening here. Something like

> 1. Return the String value consisting of the concatenation of the result of applying UTF16Encoding to each code point of _func_.[[SourceText]].

Reply:
@jmdyck This comment is not addressed.

Reply:
That's because I'm waiting for #1552 to be merged. Once that happens, the step in question will be just:
```1. Return ! UTF16Encode(_func_.[[SourceText]]).```
and the definition of `UTF16Encode` will call out the concatenation as you request.